getting-started-tutorial/tutorial-1-MLRun-basics.ipynb

```diff
@@ -0,0 +1,1890 @@
+{
```

Comment:
<p>typo: <code>function</code> ﻿.</p><p> </p><p>Also, function registration was done above, when we <code>project.set_function(gen_data_func)</code>  . </p><p>When saying the function code is only saved in a temp folder, are you talking about:</p><ul><li>The RUN is not logged?</li><li>The function under project/functions is not logged</li><li>The function "connected" from the run itself is not available?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demos/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>